docs/galapagos_user_guide.md

```diff
@@ -0,0 +1,460 @@
+   the environment) valid for 1 or 2 years. The previous certificate can still be used (as long as it
+   is valid), just exchange it at the next opportunity.
+   
+### Replace certificate
+
+   If you have lost the private key or have to recreate the client certificate for any other reasons, you can proceed in exactly the same way as when re-issuing the 
+   certificate (but not as with the renewal). Please note that the previous certificate immediately
+   loses its validity when the new certificate is generated! If the application is already running 
+   in a prod-environment with the private key and you "just" can no longer access it, you could kill your 
+   application by generating a new certificate! Galapagos will warn you when trying to generate a new certificate, if you already 
+   have one:
+   
+   ![](galapagos_user_guide_images/cert-warn.png)
+   
+## Create developer certificate   
+       
+   Depending on your team organization and deployment strategy, not every developer should have the private key of 
+   your application - especially not for production. You may still have to be able to look 
+   at the topics during production, for example. There are tools for this, but of course these also require a client
+   certificate (and the trust store) for the connection to connect to Kafka. 
+   
+   Therefore, you can get a developer certificate with Galapagos. This basically grants you the same rights as the
+   applications of which you are the Topic Administrator ("My Applications"), but are always very limited in time.  
+   Currently the times are 90 days on DEV / 
+   , 30 days on INT and 10 days on PROD. Then you have to create a new
+   certificate again - there is no extension.   
+   
+   To create a developer certificate, click on your name in the top right corner in Galapagos. Now select
+   "User Settings" in the menu. The following screen opens:
+   
+   ![](galapagos_user_guide_images/dev-cert.png)
+   
+   Select an environment for which you would like to get a developer certificate. If you had already created a certificate for that environment,
+   it becomes invalid immediately after the creation of a new certificate.
+   
+   Click on "Generate developer certificate" and you will receive a .p12 file for download. The file-
+   password is "changeit".
+   Your username is encoded in the certificate - so don't grant others access to the certificate.
+   
+## Create internal Topic   
+   
+   Internal Topics can be used for the communication between different components of your application.
+   You can use Galapagos to create an internal Topic: Go to "Create topic" via the menu on
+   the left. Select the topic type "App internal". Select your owning application and the target environment (the direct creation of topics is only 
+   allowed on the DEV / TEST environment with Galapagos!). Galapagos automatically generates a
```

Comment:
Hermes specific information. Perhaps we should call the first environment just "DEV" in the user manual, and put some information at the beginning of the manual, that we assume a setup with three environments, DEV, INT, and PROD.